src/routes/searchPhrases.js

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+let router = express.Router();
+import knex from "../knex";
+router.get('/', (req, res) => listWrapper(req, res));
+
+const listWrapper = async (req, res) => {
+    const search = req.query.search;
+
+    if (search.length > 128) {
+        console.log("search length: ", search.length);
+        return res.status(400).json({success: false, error: 'Bad Request - search term is too long (max. 128 characters)'});
+    }
+
+    const city = req.query.city;
+    const language = req.query.language;
+
+    let item = await createQuery("phrase", "search_phrase", city, search, language);
+
+    let result = {
+        item
+    }
+
+    return res.status(200).json({success: true, phrases: result});
```

Comment:
Currently, the response looks like this:
```
{
    "success": true,
    "phrases": {
        "item": [
            {
                "suggestion": "wienerwald"
            },
            {
                "suggestion": "Wienerwald"
            }
        ]
    }
}
```

It would be sufficent to have some layers less, something like:
```
{
   "success": true,
    items: [
      "wienerwald", "Wienerwald"
    ]
}
```
or (in case the endpoint will be extended with more fields in the future)
```
{
 "success": true,
  "items": [
      {
          "suggestion": "wienerwald"
      },
      {
          "suggestion": "Wienerwald"
      }
  ]
}
```

The FE would need to be adapted to a new mapping as well.